src/AtomShell/window.jl

```diff
@@ -154,71 +163,116 @@ flashframe(win::Window, on = true) =
   @dot_ win flashFrame($on)
 
 """
-    progress(win::Window, p=-1)
+    progress!(win::Window, p=-1)
```

Comment:
It looks like you changed `win::Window` to `window` inconsistently in the following docstrings. Would you be able to go back and make them all consistent? Sorry. 😬 

I'm fine with either; up to you.  I see the appeal of `win::Window`, though: it really does work for any other types, and these aren't meant to be generic functions.